packages/util/src/bn/toU8a.ts

```diff
@@ -23,25 +27,36 @@ import bnToHex from './toHex';
  * bnToU8a(new BN(0x1234)); // => [0x12, 0x34]
  * ```
  */
-export default function bnToU8a (value: BN | number, bitLength: number, isLe: boolean): Uint8Array {
-  const byteLength = Math.ceil(bitLength / 8);
+export default function bnToU8a (value: BN | number | null, options?: Options): Uint8Array;
```

Comment:
We will need to come back and only keep this - first I will go through the code and cleanup to new version, then we can drop. It just becomes messy. (I am 100% with you that iw was needed, we just need to cleanup ASAP - will log and start when bored)